platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapView.java

```diff
@@ -382,6 +382,10 @@ public void onDestroy() {
 
   @Override
   public boolean onTouchEvent(MotionEvent event) {
+    if (nativeMapView == null) {
```

Comment:
What about extracting this block of code into a `private` method? IMO, we should try to keep `public` methods as clean as possible so it reads like pseudocode (a.k.a. no conditionals, `for`s, etc.).